examples/api-tests/src/file-search.spec.js

```diff
@@ -14,10 +14,12 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
+const { ConsoleOptions } = require('@theia/console/lib/browser/console-widget');
```

Comment:
The import statement should be within the `describe`, after:

```ts
const { QuickFileOpenService } = require('@theia/file-search/lib/browser/quick-file-open');
```

Reply:
Actually, we don't need that import statement, I will remove it from the code